app/db.py

```diff
@@ -14,7 +14,7 @@ def add_student(name: str, email: str):
     cursor.execute(sql, (name, email))
     data = {'id': connection.insert_id(), 'name': name, 'email': email}
     connection.commit()
-    return data
+    return data or 'DB table is empty'
```

Comment:
We should maintain the datatype otherwise the Front-End will fail.
The data is of type list so you should do:

return data or []

This way FE will be functional.

Great Job @dshishir13 , please go-ahead and try in FE and then correct as i suggested. Then update this MR.

Reply:
sure sir thanks for the info

Reply:
done